beyond-essence/inventory/error-messages.md

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
@ywolpers Is there a reason for not including the above images in this text? If you don't want to include them anymore, you should remove them from this PR :)